kube-dns.yaml-e

```diff
@@ -0,0 +1,210 @@
+# Copyright 2016 The Kubernetes Authors.
```

Comment:
What is this file? Remove?

Reply:
Yeah, coming from rebasing I think. I will remove.